packages/expo-modules-core/package.json

```diff
@@ -8,6 +8,26 @@
     "*.fx.ts",
     "./src/polyfill/*.ts"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./build/index.d.ts",
+      "default": "./src/index.ts"
+    },
+    "./types": "./types.d.ts",
+    "./src/Refs": {
+      "types": "./build/Refs.d.ts",
+      "default": "./src/Refs.ts"
+    },
+    "./src/polyfill/dangerous-internal": {
+      "types": "./build/polyfill/dangerous-internal.d.ts",
+      "default": "./src/polyfill/dangerous-internal.ts"
+    },
+    "./src/uuid": {
+      "types": "./build/uuid/index.d.ts",
+      "default": "./src/uuid/index.ts"
+    }
```

Comment:
I'd still like to remove "src" from the API. Is the issue that we have two bundlers - one that supports package.json exports and one that doesn't - that both bundle code that imports `'expo-modules-core/src/uuid'`, and so the the package.json export names must match file names?